client/profile-wizard/index.js

```diff
@@ -215,6 +221,32 @@ class ProfileWizard extends Component {
 		}
 	}
 
+	skipProfiler() {
+		const { createNotice, updateProfileItems } = this.props;
+		updateProfileItems( {
+			skipped: true,
+			step: this.getCurrentStep().key,
+		} )
+			.then( ( response ) => {
+				if ( response.status === 'success' ) {
+					recordEvent( 'storeprofiler_store_details_skip' );
+					const href = getAdminLink( 'admin.php?page=wc-admin' );
+					getHistory().push( href );
+				} else {
+					throw new Error();
```

Comment:
Are there any details in `response` when it fails that we could log? I'm trying to think about how an HE or dev could troubleshoot things if this situation was encountered. What are the most likely reasons why this would fail?

Reply:
Hi, thank you for your suggestions. I removed the `throw` in the commit bec1b23